.circleci/config.yml

```diff
@@ -1,4 +1,4 @@
-version: 2
+version: 2.1
```

Comment:
What is the version `2.1` necessary for?

Reply:
We are using [circleci orbs](https://circleci.com/docs/2.0/orb-intro/) (line 105). In the tutorial, they used version `2.1`,  but I'm not sure if it is required.

Reply:
Could you test if it also works with version `2` ?

Reply:
https://circleci.com/docs/2.0/configuration-reference/#orbs-requires-version-21